test/spec/modules/discoveryBidAdapter_spec.js

```diff
@@ -137,4 +169,42 @@ describe('discovery:BidAdapterTests', function () {
     expect(bid.height).to.equal(250);
     expect(bid.currency).to.equal('USD');
   });
+
+  describe('discovery: getUserSyncs', function() {
+    const COOKY_SYNC_IFRAME_URL = 'https://asset.popin.cc/js/cookieSync.html';
+    const IFRAME_ENABLED = {
+      iframeEnabled: true,
+      pixelEnabled: false,
+    };
+    const IFRAME_DISABLED = {
+      iframeEnabled: false,
+      pixelEnabled: false,
+    };
+    const GDPR_CONSENT = {
+      consentString: 'gdprConsentString',
+      gdprApplies: true
+    };
+    const UPS_CONSENT = {
```

Comment:
typo, should be `USP_CONSENT`. needs updates throughout the spec file

Reply:
Hi, i have fix it.